CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 **New Features and Enhancements:**
 
+- Output contents of array for bulk commands ([#217](https://github.com/box/boxcli/pull/217))
+
 **Bug Fixes:**
 
 - Fix bug with setting proxy settings ([#218](https://github.com/box/boxcli/pull/218))
```

Comment:
```suggestion
- Output contents of array for bulk commands ([#217](https://github.com/box/boxcli/pull/217))

**Bug Fixes:**

- Fix bug with setting proxy settings (#218)
```

Reply:
This notation is all that is needed.  Let's omit the URLs, going forward.  It resolves properly at https://developer.box.com/changelog/ this way, too.